src/platform/site-wide/side-nav/components/SideNav.js

```diff
@@ -52,11 +56,12 @@ class SideNav extends Component {
     // Derive the items to render.
     const filteredNavItems = filter(
       navItemsLookup,
+      // eslint-disable-next-line lodash/matches-prop-shorthand,lodash/matches-shorthand
```

Comment:
Why is this being disabled?

Reply:
I'm not sure, it looks like something that got ported over when I was splitting the larger PR. @bradl3yC was there a reason why you disabled this?

Reply:
This is probably due to a configuration item on the eslint config.  Its throwing a warning because we're accessing a prop on the first object